docs/advanced_guide/cloud/notebooks/text_generation/text_generation.ipynb

```diff
@@ -0,0 +1,469 @@
+        "display(latency_vs_memory_img)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "#### Show search state of the last iteration"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "df = nb_helper.get_search_csv(downloaded_folder)\n",
+        "csv_as_html = nb_helper.get_csv_as_stylized_html(df)\n",
+        "display(HTML(csv_as_html))"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "### Job 2: Train (Train a Pareto architecture from Transformer-Flex.)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "#### Pick an architecture id (archid) from the CSV file to perform full training on"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "archid = \"<arch-id>\"\n",
+        "arch_path = nb_helper.get_arch_abs_path(archid=archid, downloaded_folder=downloaded_folder)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "#### Load the training job from a YAML file and set the arch_path as its input"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "train_job = load_job(source=os.path.join(\"src\", \"train.yaml\"))\n",
+        "train_job.inputs.pareto_config_path.path = arch_path"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "t_job = ml_client.create_or_update(train_job)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "#### Stream logs of the job"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "ml_client.jobs.stream(t_job.name)"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "### Job 3: Generating text via prompt"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "#### Load the generate text job from a YAML file, set the trained model path, and run it"
```

Comment:
remove '####' and can we figure out this path programatically so the user doesn't have to?

Reply:
I'll change this Notebook to make it an AML Pipeline, then the user won't have to set up this path.

Reply:
Done.